odxtools/cli/_print_utils.py

```diff
@@ -1,9 +1,29 @@
 # SPDX-License-Identifier: MIT
 import re
+from typing import Any, Dict, List, Optional, Union
 
 import markdownify
+from tabulate import tabulate
 
+from ..diaglayer import DiagLayer
 from ..diagservice import DiagService
+from ..parameters.codedconstparameter import CodedConstParameter
+from ..parameters.nrcconstparameter import NrcConstParameter
+from ..parameters.parameter import Parameter
+from ..parameters.physicalconstantparameter import PhysicalConstantParameter
+from ..parameters.systemparameter import SystemParameter
+from ..parameters.valueparameter import ValueParameter
+from ..singleecujob import SingleEcuJob
+
+
+class Style:
+    # class with variables and functions to display the result of the comparison
+
+    ## text style variables
+    bold = '\033[1m'
+    underline = '\033[4m'
+    italic = '033[3m'  # doesn't work with Windows CMD and PowerShell
```

Comment:
Normally I would recommend using already established python modules that handle coloring, detect if console does not support color, etc.

See https://pypi.org/project/termcolor/ or https://pypi.org/project/rich/

Reply:
text formatting in the terminal is done via the python module `rich` now